src/getOperands.js

```diff
@@ -27,6 +27,8 @@
  */
 function getOperands(str) {
   // write code here
+  let operands = str.match(/-?\d(\.\d+)?/g);
```

Comment:
prefer const
this regExp do not match these numbers `23.5 - 987.32` because you match only one digit before point

Reply:
now it works )